srcpkgs/kleopatra/template

```diff
@@ -0,0 +1,18 @@
+# Template file for 'kleopatra'
+pkgname=kleopatra
+version=20.12.3
+revision=1
+build_style=cmake
+hostmakedepends="extra-cmake-modules kdoctools qt5-qmake qt5-host-tools kcoreaddons gettext kconfig"
+makedepends="kcmutils-devel AppStream kitemmodels-devel gpgmeqt-devel libkleo-devel kmime-devel"
+depends="hicolor-icon-theme libkleo kcmutils kmime gnupg2 gnupg2-scdaemon pinentry-qt"
```

Comment:
```suggestion
depends="gnupg2 gnupg2-scdaemon pinentry-qt"
```
Is the smartcard daemon actually a hard dependency? This sounds more like smth that users just should install if they use smartcards?

The Kde libs are auto detected by xbps-src, no need to manually specify them. (See `04-generate-runtime-deps` stage in the logs)
`hicolor-icon-theme` is also not a really required dependecy, you just need some icons, but xbps doesn't have the ability to express that.